src/main/java/uk/gov/pay/api/model/PaymentWithSelfLinksJSON.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.pay.api.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(description = "A Payment with only a self link")
+public interface PaymentWithSelfLinksJSON {
```

Comment:
This is the same as PaymentWithLinksJSON with the exception of the `@ApiModelProperty`.
We can sit down and find a better name convention or implementation..